update_product_stream/README.md

```diff
@@ -13,6 +13,13 @@ After the RPM has been installed, the documentation will be available at `/usr/s
 - [Check for latest documentation](#check-for-latest-documentation)
 - [Check for field notices about hotfixes](#check-for-field-notices-about-hotfixes)
 
+## Optional prerequisites
+This step should ONLY be performed if an http proxy is required to access a public endpoint on the internet for the purpose of downloading artifacts. CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints. The http proxy variables must NOT be exported and must be `unset` after the desired artifacts are downloaded. Failure to unset the http proxy variables after downloading artifacts will cause many failures in subsuquent install steps.
```

Comment:
This is a common procedure used in installs and upgrades so may want to add upgrades at the end:
"failures in subsequent install or upgrade steps"